xrpc/xrpc.go

```diff
@@ -45,12 +45,16 @@ const (
 )
 
 func makeParams(p map[string]interface{}) string {
-	var parts []string
+	params := url.Values{}
 	for k, v := range p {
-		parts = append(parts, fmt.Sprintf("%s=%s", k, url.QueryEscape(fmt.Sprint(v))))
+		if s, ok := v.([]string); ok {
+			params.Add(k, strings.Join(s, ","))
```

Comment:
so we want `p=x,y,z` instead of `p=x&p=y&p=z`? Can you at least document that on `func makeParams`, about what the values can be and what it does, while you're here?

Also, might as well change `interface{}` to `any` since this module requires Go 1.20+ anyway.

Reply:
The case I was getting at is we want `p=x,y,z` instead of `p=[x+y+z]` which is how it currently handles slices of strings. I'll try converting this to use `any`.

Reply:
Added a comment clarifying behavior.